Microsoft.DotNet.Try.Jupyter/JupyterRequestContextHandler.cs

```diff
@@ -30,155 +35,190 @@ public JupyterRequestContextHandler(PackageRegistry packageRegistry)
-                            delivery.Command.Request.Header);
-
-                        executeReply.Identifiers = delivery.Command.Request.Identifiers;
-
-                        serverChannel.Send(executeReply);
-                    }
-                    else
-                    {
-                        var errorContent = new Error
-                        {
-                            EName = string.IsNullOrWhiteSpace(result.Exception)
-                                        ? "Compile Error" 
-                                        : "Unhandled Exception",
-                            EValue = output,
-                            Traceback = new List<string>()
-                        };
-
-                        //  reply Error
-                        var executeReplyPayload = new ExecuteReplyError(errorContent)
-                        {
-                            ExecutionCount = _executionCount
-                        };
-
-                        // send to server
-                        var executeReply = messageBuilder.CreateMessage(
-                            MessageTypeValues.ExecuteReply, 
-                            executeReplyPayload, 
-                            delivery.Command.Request.Header);
-
-                        executeReply.Identifiers = delivery.Command.Request.Identifiers;
-
-                        serverChannel.Send(executeReply);
-
-                        if (!executeRequest.Silent)
-                        {
-                            // send on io
-                            var error = messageBuilder.CreateMessage(
-                                MessageTypeValues.Error, 
-                                errorContent, 
-                                delivery.Command.Request.Header);
-                            ioPubChannel.Send(error);
-
-                            // send on stderr
-                            var stdErr = new StdErrStream
-                            {
-                                Text = errorContent.EValue
-                            };
-                            var stream = messageBuilder.CreateMessage(
-                                MessageTypeValues.Stream, 
-                                stdErr, 
-                                delivery.Command.Request.Header);
-                            ioPubChannel.Send(stream);
-                        }
-                    }
-
-                    if (!executeRequest.Silent && resultSucceeded)
+            // executeResult data
+            var executeResultData = new ExecuteResult
+            {
+                Data = new JObject
+                {
+                    {"text/html", output},
+                    {"text/plain", output}
+                },
+                Transient = transient,
+                ExecutionCount = _executionCount
+            };
+
+
+            var resultSucceeded = result.Succeeded &&
+                                  result.Exception == null;
```

Comment:
Are we (somewhere) constraining this so that `Succeeded` must be false when there's an `Exception`? The possibility that we would have `Succeeded == true` and an exception looks wrong.

Reply:
at [RoslynWorkspaceServer.cs](https://github.com/colombod/try/blob/e22606aff72146dfa7a41113d83523e049350a52/WorkspaceServer/Servers/Roslyn/RoslynWorkspaceServer.cs#L338)

we can have 

` var runResult = new RunResult(
                succeeded: true,
                output: output.StdOut,
                exception: exceptionMessage,
                diagnostics: diagnostics,
                requestId: requestId);`

with `exceptionMessage` not null